flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -180,6 +180,22 @@
 			// default matches heartbeat.timeout so that sticky allocation is not lost on timeouts for local recovery
 			.defaultValue(HeartbeatManagerOptions.HEARTBEAT_TIMEOUT.defaultValue())
 			.withDescription("The timeout in milliseconds for a idle slot in Slot Pool.");
+
+	/**
+	 * Hadoop job history server URL.
+	 */
+	public static final ConfigOption<String> JOB_MANAGER_HADOOP_HISTORYSERVER_URL =
```

Comment:
instead of having the JobManagerOptions to load the hadoop history server logs, I think it makes more sense to have it appeared in `HistoryServer` - Currently Flink Session cluster UI already have ways to show JM/TM logs I was not sure if this is needed for the online Flink UI.